### ISSUE TITLE
If the Input is in MatFormField, we should set the width to MatFormField

### ISSUE THREAD

**Issue by bthieblot** [DesertFoxNV/ngx-autosize-input#11]
Hello,

Me again, like in the title, if we use the Input inside a MatFormField tag, we should set the width to the MatFormField and not in the input..

But I don't know if it's the purpose of your package.. If you want to apply only on input or what ! Thanks

Tell me, PR is coming !

**Comment by joshuawwright** [comment]
Originally, I had only planned to set it to the input. However, extending it to apply to another element would be quite useful.

**Comment by joshuawwright** [comment]
This should now be resolved in version 2.3.

**Comment by joshuawwright** [comment]
@bthieblot I ended up making this more generic. The parameter "setParentWidth", when true updates the width of the parent.

```
<mat-form-field> // This will be resized
    <input AutoSizeInput [setParentWidth]="true">
</mat-form-field>
```
